cmd/meroxa/turbine_cli/python/deploy.go

```diff
@@ -87,16 +88,32 @@ func GetResourceNames(ctx context.Context, l log.Logger, appPath, appName string
 	if err != nil {
 		return names, errors.New(string(output))
 	}
-	r := regexp.MustCompile("^turbine-response: \\[(.*)\\]\n")
-	match := r.FindStringSubmatch(string(output))
-	if match == nil || len(match) < 2 {
-		return names, fmt.Errorf("unable to verify resource availability for Meroxa Application at %s; %s", appPath, string(output))
+
+	var parsed []map[string]interface{}
```

Comment:
what does this data actually look like? i'm a little concerned about this data type

Reply:
The listresources response looks like this: 

[ { "name": "source", "source": true, "destination": false, "collection": "orders" }, { "name": "dest", "source": false, "destination": true, "collection": "test" } ]

It is now the same format as turbine-go response , the code for parsing it is essentially the same now as turbine go too

Reply:
@anna-cross we can create a struct type to formalize it, this will help us decode what we expect

Reply:
@janelletavares likely came from here https://github.com/meroxa/cli/blob/master/cmd/meroxa/turbine_cli/golang/deploy.go#L52-L56

i think once we have this in place we can refactor it to be stricter.

Reply:
+1 would prefer a []struct for the unmarshling pointer

Reply:
Updated to use a struct :D will do the same for JS